lib/actions/os.coffee

```diff
@@ -43,9 +94,26 @@ exports.download =
 
 		console.info("Getting device operating system for #{params.type}")
 
-		manager.get(params.type).then (stream) ->
-			bar = new visuals.Progress('Downloading Device OS')
-			spinner = new visuals.Spinner('Downloading Device OS (size unknown)')
+		version = options.version
+		displayVersion = ''
+		Promise.try ->
+			if not version
+				version = 'default'
+				console.warn('OS version is not specified, using the default version:
+					the newest stable (non-pre-release) version if available,
+					or the newest version otherwise (if all existing
+					versions for the given device type are pre-release)')
+				return
+			return resolveVersion(params.type, version)
+		.then (resolvedVersion) ->
+			version = resolvedVersion
```

Comment:
Oh, that's the mistake I made when refactoring the code. 'default' should be returned above